db/db.go

```diff
@@ -3,7 +3,7 @@ package db
 import (
 	"github.com/streamingfast/kvdb/store"
 	"github.com/streamingfast/logging"
-	kvv1 "github.com/streamingfast/substreams-sink-kv/pb/substreams/sink/kv/v1"
```

Comment:
@maoueh Should I leave these untouched then? These were already named `kvv1`.

Reply:
Ok quite interesting hehe. Yes let's be consistent and use `pbkv` for now. I'll research a bit more.